<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Axis3DDock</class>
 <widget class="QWidget" name="Axis3DDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>737</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAxes">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="dsbMinRange"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="leXLabel"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="maxRangeLabel">
         <property name="text">
          <string>Max Range</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="segmentCountLabel">
         <property name="text">
          <string>Segment Count</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="xLabel">
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="subSegmentCountLabel">
         <property name="text">
          <string>Sub-segment Count</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QSpinBox" name="sbSegmentCount"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="xAxisFormatLabel">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="chkVisible">
         <property name="text">
          <string>Visible</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QDoubleSpinBox" name="dsbMaxRange"/>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>272</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="cbXLabelFormat"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="minRangeLabel">
         <property name="text">
          <string>Min Range</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QSpinBox" name="sbSubSegmentCount"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>leXLabel</tabstop>
  <tabstop>cbXLabelFormat</tabstop>
  <tabstop>sbSegmentCount</tabstop>
  <tabstop>sbSubSegmentCount</tabstop>
  <tabstop>dsbMinRange</tabstop>
  <tabstop>dsbMaxRange</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
